proxy.go

```diff
@@ -311,6 +311,16 @@ func (rp *reverseProxy) proxyRequest(s *scope, rw ResponseWriterWithCode, srw *s
 		proxiedResponseDuration.With(s.labels).Observe(duration)
 	}, func(labels prometheus.Labels) { retryRequest.With(labels).Inc() })
 
+	statusCodesProxy.With(
+		prometheus.Labels{
+			"user":         s.user.name,
```

Comment:
do you need the user & cluster_user in the labels? Because more cardinality means more memory used to store the state and slower code.

Reply:
deleted used & cluster_user in the labels https://github.com/ContentSquare/chproxy/pull/407/commits/c3ebc353912d8c4f5544a6e25fcf09b24ceddd3c